File: pizzayolo/src/main/java/pizzayolo/entity/Dessert.java
package pizzayolo.entity;

public class Dessert {

	private int id;
	private String nom;
	private double prix;

	
	
	public Dessert(String nom, double prix) {
		this.nom = nom;
		this.prix = prix;
	}
		
	
	
	public Dessert(int id, String nom, double prix) {

		this.id = id;
		this.nom = nom;
		this.prix = prix;
	}
	
	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getNom() {
		return nom;
	}

	public void setNom(String nom) {
		this.nom = nom;
	}

	public double getPrix() {
		return prix;
	}

	public void setPrix(double prix) {
		this.prix = prix;
	}



	@Override
	public String toString() {
		return "Dessert [id=" + id + ", nom=" + nom + ", prix=" + prix + "]";
	}

}

File: pizzayoloangular/src/app/model/dessert.ts
export class Dessert {
  public constructor(
    private _nom?: string,
    private _prix?: number,
    private _id?: number
  ) {}

  get nom(): string | undefined {
    return this._nom;
  }

  set nom(value: string | undefined) {
    this._nom = value;
  }

  get prix(): number | undefined {
    return this._prix;
  }

  set prix(value: number | undefined) {
    this._prix = value;
  }
  get id(): number | undefined {
    return this._prix;
  }

  set id(value: number | undefined) {
    this._prix = value;
  }
}
